modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -193,7 +228,7 @@ private string DetermineRuntimeIdentifierArch(string arch)
                 "x86_64" => "x64",
                 "armeabi-v7a" => "arm",
                 "arm64-v8a" => "arm64",
-                "armv7" => "arm",
+                "arm32" => "arm",
```

Comment:
This seems like an unrelated change that could've been a separate PR. I'm guessing we just missed it because we weren't supporting this architecture before. Seems to be a result of https://github.com/godotengine/godot/pull/55778.

Reply:
Could be a different PR, yes. but it doesn't appear to be related to that PR specifically.

The keys are what ends up in the data directly name and must match https://github.com/godotengine/godot/blob/master/core/config/engine.cpp#L196-L230 to load correctly.

Reply:
Since `arm32` support seems broken currently, might be worth re-assessing whether we're doing everything right with these remappings to see if the problem is in Godot or .NET.

Reply:
This remap should result in the runtime ID `android-arm` which is correct AFAIK: https://github.com/dotnet/runtime/blob/25c675ff78e0446fe596cea25c7e3969b0936a33/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json#L238

However, the package `Microsoft.NETCore.App.Runtime.linux-bionic-arm` does not seem to exist, so it seems like it's just unsupported in .NET. The runtime ID `android-arm` is also not listed in the [Known RIDs](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#known-rids).